kategory-core/src/main/kotlin/kategory/data/StateT.kt

```diff
@@ -5,7 +5,8 @@ typealias StateTFunKind<F, S, A> = HK<F, StateTFun<F, S, A>>
 
 fun <F, S, A> StateTKind<F, S, A>.runM(initial: S): HK<F, Tuple2<S, A>> = (this as StateT<F, S, A>).run(initial)
 
-@higherkind class StateT<F, S, A>(
+@higherkind
+class StateT<F, S, A>(
```

Comment:
The ones currently in master merged in #264 already include the refactoring for StateT that is consistent in naming conventions and implementations with the rest of the code base. You can accept `master` changes when updating this branch.